### ISSUE TITLE
Get wrong size when Translucent Status Bar

### ISSUE THREAD

**Issue by rexi1r** [ankushsachdeva/emojicon#15]
hi.
when set status bar translucent the size of popup is wrong .how can i fix this issue?

**Comment by rexi1r** [comment]
Did you check it?
when translucent status bar in android L the size of popup view is bigger than keyboard height.
when translucent status bar in android 4.4.2, when i click on EditText, emoji_btn Do not move to the top of keyboard.
![screenshot_2015-03-29-10-15-25](https://cloud.githubusercontent.com/assets/4881902/6884266/9226a7d6-d5fd-11e4-9590-7fe46d4f60cc.png)

**Comment by rexi1r** [comment]
I've solved the problem with the following change:

``` java
      if (!((Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) && (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP))) {
                    if (resourceId > 0) {
                        heightDifference -= mContext.getResources()
                                .getDimensionPixelSize(resourceId);
                    }
                }
```

**Comment by ankushsachdeva** [comment]
Yes you are right, for translucent status bar, its height should not be subtracted from the heightDifference.
I'll add a flag for people using translucent status bar in their apps or if anyone could suggest any better solution for this and similar problems.

**Comment by rexi1r** [comment]
the above change work, in both situation( with translucent status bar or without translucent )

**Comment by cridus** [comment]
I just tried KitKat in an emulator, with a non-translucent status bar and the popup is of the right height without the above change to the code (so, with the status bar's height being subtracted).
